packages/admin-panel/src/dataTables/components/PreviewFilters/filters/OrganisationUnitCodesField.js

```diff
@@ -3,25 +3,19 @@
  * Copyright (c) 2017 - 2023 Beyond Essential Systems Pty Ltd
  */
 
-import React, { useEffect, useState } from 'react';
+import React, { useState } from 'react';
 
 import PropTypes from 'prop-types';
 
 import { ParameterType } from '../../editing';
-import { useLocations } from '../../../../VizBuilderApp/api';
+import { useEntities } from '../../../../VizBuilderApp/api';
 import { Autocomplete } from '../../../../autocomplete';
 
-export const OrganisationUnitCodesField = ({ name, onChange, runtimeParams }) => {
```

Comment:
can you also remove `runtimeParams` on where it is called? It is at `PreviewFilters.js`.